selfdrive/car/toyota/values.py

```diff
@@ -258,6 +267,7 @@ class CAR:
   CAR.RAV4: dbc_dict('toyota_rav4_2017_pt_generated', 'toyota_adas'),
   CAR.PRIUS: dbc_dict('toyota_prius_2017_pt_generated', 'toyota_adas'),
   CAR.COROLLA: dbc_dict('toyota_corolla_2017_pt_generated', 'toyota_adas'),
+  CAR.LEXUS_RX: dbc_dict('LEXUS_RX_2016_pt_generated', 'toyota_adas'),
```

Comment:
Does it work with the DBC name being uppercase?

Reply:
Oops, that was a find and replace mistake. Fixed